tool/debian/singa-cpp_1.0.0_amd64/DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Package: Singa
+Version: 1.0.0
+Architecture: amd64
+Maintainer: Moaz Reyad <moazreyad@gmail.com>
+Depends: libgoogle-glog-dev, libprotobuf-dev, libstdc++6 , libc6
```

Comment:
how about openblas?
I think we need both libprotobuf and libopenblas.
Would apt-get install the 'Depends' automatically if they are not available in the system?